src/WebJobs.Script/Environment/EnvironmentSettingNames.cs

```diff
@@ -89,6 +89,7 @@ public static class EnvironmentSettingNames
         public const string PodEncryptionKey = "POD_ENCRYPTION_KEY";
         public const string HttpLeaderEndpoint = "HTTP_LEADER_ENDPOINT";
         public const string ManagedEnvironment = "MANAGED_ENVIRONMENT";
+        public const string ConnectedEnvironment = "CONNECTED_ENVIRONMENT";
```

Comment:
Do we know if this env variable will actually exist in case of connected env?

Reply:
Should we be more verbose with naming of this environment variable?

Reply:
We would need to add this from ACA RP.

Reply:
The name verbally seems confusing because there is Environment in the name which refers to infrastructure env of the containerApp.